src/main/java/it/gov/pagopa/ranking/service/RankingRequestsApiServiceImpl.java

```diff
@@ -42,7 +44,7 @@ public List<RankingRequestsApiDTO> findByInitiativeId(String organizationId, Str
                 List<OnboardingRankingRequests> requests =
                         onboardingRankingRequestsRepository.findByInitiativeId(
                                 initiativeId,
-                                PageRequest.of(page, size, Sort.by("rank"))
+                                PageRequest.of(page, size, Sort.by("ranking"))
```

Comment:
let's use the OnboardingRankingRequest.Fields.rank constant provided by lombok annotating the model with @FieldNameConstants